extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -283,13 +293,19 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       // dimensions in an array.
       const dimensions = getMultiSizeDimensions(
           multiSizeDataStr,
-          Number(this.element.getAttribute('width')),
-          Number(this.element.getAttribute('height')),
-          multiSizeValidation == 'true');
+          this.size_.width, this.size_.height,
+          multiSizeValidation == 'true',
+          /* Use strict mode only in non-refresh case */ !this.isRefreshing);
       sizeStr += '|' + dimensions
           .map(dimension => dimension.join('x'))
           .join('|');
     }
+    const rc = (this.refreshCount_ && this.fromResumeCallback)
+        ? this.refreshCount_ + 1
+        : (this.fromResumeCallback ? 1 : this.refreshCount_ || null);
+    this.win['ampAdGoogleIfiCounter'] = this.win['ampAdGoogleIfiCounter'] || 1;
+    this.ifi_ = (this.isRefreshing && this.ifi_) ||
```

Comment:
On the first pass `this.ifi_` will be set to 1. On the second pass, if `this.isRefreshing` is true, then we'll use the value of `this.ifi_` (no incrementing), otherwise we'll use the incremented value of `this.win['ampAdGoogleIfiCounter']`.